tests/unit/components/UI/Alert/DialogBox.spec.js

```diff
@@ -11,19 +15,66 @@ describe("DialogBox.vue", () => {
     const title = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        title,
+        title: title,
       },
     });
-    expect(wrapper.find("p").text()).toBe(title);
+
+    expect(wrapper.get('[data-test="title"]').text()).toBe(title);
   });
 
   it("renders description correctly", () => {
     const description = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        description,
+        description: description,
+      },
+    });
+
+    expect(wrapper.get('[data-test="description"]').text()).toBe(description);
+  });
+
+  it("renders image correctly", () => {
+    const wrapper = mount(DialogBox, {
+      props: {
+        iconConfig: {
+          enabled: true,
+          name: "check",
+          class: "w-12 h-12",
+        },
+      },
+      global: {
+        stubs: {
+          InlineSvg: InlineSvg,
+        },
       },
     });
-    expect(wrapper.find("p").text()).toBe(description);
+    expect(wrapper.get('[data-test="icon"]').html()).toContain("img");
+    expect(wrapper.get('[data-test="icon"]').classes()).toEqual(
+      expect.arrayContaining(["w-12", "h-12"])
+    );
+  });
+
+  it("confirm button present", () => {
+    const wrapper = mount(DialogBox);
+    expect(wrapper.get('[data-test="confirmButton"]').text()).toBe("Yes");
```

Comment:
Isn't it best to just check if the button element exists instead of checking the text inside the button? The button text might change in the future and also if checking for text, we have to make sure that all corresponding i18n strings also match